low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -203,15 +206,17 @@ class SSPLDevDeploy:
                                   out,
                                   "Failed to install %s." % (pkg_name),
                                   cmd)
-
-            print("INFO: INSTALLING rabbitmq-server...")
-            cmd = "yum install -y rabbitmq-server"
+            # Install cortx-py-utils dependencies
+            cmd = ("pip3 install -r "
```

Comment:
all 3rd party requirements from all cortx components are being consolidated by RE in one pkg, after that this file may not be available. Check with Provisioning & RE team about this approach & what way can be future proof.

Mail from RE regarding same, with subject "DevOps: Install Third party packages (Python and RPM)"
>cortx-prereq.rpm is available with main branch now.  Both Python and RPM package list’s are available in cortx-re repo at below location.
Python dependencies - https://github.com/Seagate/cortx-re/blob/main/scripts/third-party-rpm/python-requirements.txt
RPM dependencies - https://github.com/Seagate/cortx-re/blob/main/scripts/third-party-rpm/third-party-rpms.txt

Reply:
`curl -s http://cortx-storage.colo.seagate.com/releases/cortx/third-party-deps/rpm/install-cortx-prereq.sh | bash`
This script from RE will install all the required packages, I will update SSPL wiki with this  step after this is merged ( in pre-requisite steps)